### PR TITLE
Custom HeaderText on Wanted/Missing Posters

### PR DESCRIPTION
**Pull request by Rockdtben**
## About The Pull Request

This allows for this: 
image

Allows for Custom Header Text (up to 7 chars) on the wanted and missing posters.

This is the precursor to custom colors and a poster printer in library.

## Why It's Good For The Game

Precursor to ingame meme posters.

## Changelog
:cl:
add: WJohnston - Created a minimalist font.
add: Security can put custom headings on wanter/missing posters.
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
I can't wait for the clown to print a shitload of mine posters labeled "gay"

File: code/game/machinery/computer/security.dm [hunk 1]
```diff
@@ -439,29 +439,33 @@ What a mess.*/
 								default_description += "\n[c.crimeName]\n"
 								default_description += "[c.crimeDetails]\n"
 
+						var/headerText = stripped_input(usr, "Please enter Poster Heading (Max 7 Chars):", "Print Wanted Poster", "WANTED")
```
Comment (AnturK) [inline code/game/machinery/computer/security.dm]: You should set max_length param here if you know how long it's supposed to be.

**Comment by nicbn** [comment]
"BASED" looks like "BRSED" at a quick glance, you could round the A corners

**Comment by Rockdtben** [comment]
@nicbn 

image

**Comment by MrDoomBringer** [comment]
Add bottom text for true meme posters

**Comment by Rockdtben** [comment]
@MrDoomBringer Soon™

**Comment by SpinnerMaster** [comment]
> Add bottom text for true meme posters

we live in a society

**Pull request merged**